examples/readme.md

```diff
@@ -2,29 +2,37 @@ A few sample projects to help you get started with the `windows` crate.
 
 ## clock
 
-[clock](clock) - An example of using [Direct2D](https://docs.microsoft.com/en-us/windows/win32/direct2d/direct2d-overview) and various other Windows APIs.
+[clock](clock) - An example of using [Direct2D](https://docs.microsoft.com/windows/win32/direct2d/direct2d-overview) and various other Windows APIs.
 
-![clock](https://user-images.githubusercontent.com/9845234/106089513-ef5f3d80-60dc-11eb-9aee-f89d2416f341.gif)
+<img src=https://user-images.githubusercontent.com/9845234/106089513-ef5f3d80-60dc-11eb-9aee-f89d2416f341.gif width="75%">
 
-## create_window
+## com_uri
 
-[create_window](create_window) - A simple example of creating a desktop window with [CreateWindowExA](https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-createwindowexa) and friends.
+[com_uri](com_uri) - A simple example of using the [CreateUri](https://docs.microsoft.com/previous-versions/windows/internet-explorer/ie-developer/platform-apis/ms775098(v=vs.85)) function.
```

Comment:
Is this the right link to the COM API?

Reply:
Seems like this is the sad new URL for this API. @stevewhims can confirm.

Reply:
You're referring to the CreateUri function? During the migration from MSDN to docs.microsoft.com, a lot of topics were moved into the previous-versions archive. Some inadvertently. But writers can restore archived-by-mistake topics back to the fresh content on request from feature teams. If you'd like to do that for CreateUri, then let me know and I'll ping you contact details for the writer-owner via email (since I don't know whether this thread is publicly visible).

Or did you mean the `[com_uri](com_uri)` part? It's not letting me switch to Files changed, so I can't get much context around what that's meant to link to.

Reply:
Yes it was the `CreateUri` function. I don't own this so we'll just use this link as its just for an example. Thanks!